lib/open_project/custom_field_format.rb

```diff
@@ -111,6 +115,12 @@ def find_by(name:)
         registered_by_name[name.to_s]
       end
 
+      def enabled_for_class_name(class_name)
+        enabled
+          .select { |format| format.for_class_name?(class_name) && !format.label.nil? }
```

Comment:
:question: What purpose does the `label.nil?` check have? I was surprised seeing it here.